amethyst_rendy/src/lib.rs

```diff
@@ -45,8 +45,8 @@
 #![allow(clippy::new_without_default)]
 #![allow(unused_variables, dead_code)]
 
-#[macro_use]
-extern crate amethyst_derive;
+// #[macro_use]
```

Comment:
Why is this commented out? If it compiles, it should just be removed.

Reply:
This was used for prefabs which are scattered throughout the code and are commented out for the time being. Although `#[macro_use]` was deprecated since rust 1.30 so I will just remove this.